Assignment2/src/asgn2/Assignment2.java

```diff
@@ -0,0 +1,52 @@
+//Assignment 2 working on it
+package asgn2;
+
+import java.util.Scanner;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.ObjectOutputStream;
+import java.io.ObjectInputStream;
+
+public class Assignment2 {
+	public static void main(String args[]) throws Exception {
+	Scanner sc=new Scanner(System.in);
+//this program doesn't give the user choice of action as of now. The user details can only be added and displayed. 
+	System.out.println("Enter user details: ");
+	System.out.print("Name: ");
+	String name=sc.next();
+	System.out.print("Rollno: ");
+	int rollno=sc.nextInt();
+	System.out.print("Age: ");
+	int age=sc.nextInt();
+	System.out.print("Address: ");
+	String address=sc.next();
+	System.out.print("Course: ");
+	String course=sc.next();
```

Comment:
Consider restructuring this piece of code.

Reply:
Yes sir, I will use getter and setter methods instead.